spec/controllers/admin/users_controller_spec.rb

```diff
@@ -146,5 +146,27 @@ module Pageflow
         expect(user.reload.account).to eq(other_account)
       end
     end
+
+    describe '#delete_me' do
+      it 'allows to destroy the user when authorize_user_deletion true' do
+        sign_in(create(:user, password: '@qwert123'))
+
+        expect do
+          delete(:delete_me, user: {current_password: '@qwert123'})
+        end.to change { User.count }
+      end
+
+      it 'does not allow to destroy the user when authorize_user_deletion non-true' do
+        user = create(:user, password: '@qwert123')
+        account_users_count = user.account.users.length
+        sign_in(user)
+        Pageflow.config.authorize_user_deletion =
+          lambda { account_users_count > 1 ? true : 'Last user on account. Permission denied' }
```

Comment:
Normally I'd say, simply return `false` to make the test as simple as possible. On the other hand, we need to test that the block gets passed a user record. So performing an actual check inside the lambda is one way of expressing that while also demonstrating real world usage. The alternative would be to pass a mock and explicitly expect that `call` is invoked with given arguments.